lib/puppet/type/sensu_check.rb

```diff
@@ -69,6 +69,15 @@ def insync?(is)
     end
   end
 
+  newproperty(:contacts, :array_matching => :all, :parent => SensuCheckArrayProperty) do
+    desc "Contact names to override handler configuration via Contact Routing"
+    newvalues(/.*/, :absent)
```

Comment:
regex for valid contact name - https://sensuapp.org/docs/0.29/enterprise/contact-routing.html#contact-names